### PR TITLE
chore(doc): format markdown files

### PR DESCRIPTION
**Pull request by liuderchi**
- [x] `$ npm i -D glob husky lint-staged prettier`
- ~add `prettier:markdown` script~
- refine prettier config (ref https://github.com/kentcdodds/kcd-scripts/blob/main/src/config/prettierrc.js)

### CODE REVIEW

**Review by polynoman — COMMENTED** [review]
The changes to the markdown look fine. I didnt check the new depenencies. Is there anything else to look for?

**Comment by polynoman** [comment]
Ah and changelog and some other files need to be updated to match master branch status again.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,135 +4,135 @@
 
 ## Which Languages can I Contribute?
 
-  - *Any language* is welcomed. *Anyone* is welcomed to contribute. :earth_asia: :earth_americas: :earth_africa:
-  - There are several [issues](//github.com/liuderchi/atom-i18n/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) require your help.
-  - NOTE: Currently the author do not review grammar of the translation.
-
+- _Any language_ is welcomed. _Anyone_ is welcomed to contribute. :earth_asia: :earth_americas: :earth_africa:
+- There are several [issues](//github.com/liuderchi/atom-i18n/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) require your help.
+- NOTE: Currently the author do not review grammar of the translation.
```
Comment (polynoman) [inline CONTRIBUTING.md:9]: There is a grammar error, it should say:

"NOTE: Currently the author do**es** not review grammar of the translation."

Comment (liuderchi) [reply CONTRIBUTING.md]: Thanks! fixed in 9b87a88

**Pull request merged**